packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -303,7 +302,10 @@ export const messages = {
   error_no_scratch_def:
     'No scratch definition files found. These files must be in the "config" folder and have file names that end with "-scratch-def.json". See [Scratch Org Definition File](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs_def_file.htm) for help.',
   force_list_metadata: 'SFDX: Force List Metadata',
-
+  apex_execute_compile_success: 'Compiled successfully.',
```

Comment:
Are these labels the same used in the CLI command ? cc @ekapner

Reply:
Yes the `apex_execute_compile_success` and `apex_execute_runtime_success` are the same as the CLI command, but would be great if you could take a look @ekapner !
`apex_execute_text` is used for telemetry and `force_apex_execute_library` is the label we'll display to the user when running the command.

Reply:
@lcampos @AnanyaJha - a bit concerned about referencing the SFDX Dev Guide in the error message (mostly because the doc team is thinking about rearchitecting it - and I worry that this link will break - realistically we're talking 6-12 months out) - but could happen. We can take the approach to fix it once it's reported.

Reply:
@AnanyaJha - I need more context - I'll set up a meeting

Reply:
@ekapner are you referring to the `error_no_scratch_def` message? That message is actually not part of this change, it's already in production. But maybe we can create a separate work item if that link will be updated in the future

Reply:
@ekapner that's good to know. We can keep up with the status of that effort and update our links once they have a release date. From a user perspective it's valuable to have a link that takes you directly to where you need to consume the info from instead of having them search for it, huge time saver and a modern developer experience.